.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.5.4
- - 1.6.2
+ - 1.6
```

Comment:
If I'm not mistaken `1.6` means `1.6.0` (https://github.com/travis-ci/gimme/blob/master/gimme#L95)